documentation/src/main/asciidoc/topics/proc_configuring_tracing.adoc

```diff
@@ -0,0 +1,35 @@
+[id='configuring-metrics_{context}']
+= Enabling {brandname} tracing
+
+{brandname} generates tracing spans that are compatible with the OpenTelemetry open standard.
+At the time of writing tracing is defined at global level (once for all caches) passing the following system properties to the Infinispan server:
+
+.tracing.properties
+[source,options="nowrap",subs=attributes+]
+----
+include::properties/tracing.properties[]
+----
+<1> Globally enables the tracing.
+<2> The OTLP (OpenTelemetry protocol) exporter is the recommended format to export tracing data.
+<3> Specify here service name. The default name is `Infinispan Server`.
```

Comment:
@fax4ever what exactly is the default? `Infinispan Server`?

Reply:
@dvagnero yes it is